ballerina/api_cv_attachment_srv/docs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "The `ballerinax/sap.s4hana.api_del_doc_with_credit_block` package offers APIs for seamless integration with the [Delivery Document with Credit Block API v1.0.0](https://api.sap.com/api/API_DEL_DOC_WITH_CREDIT_BLOCK/overview). The service contains entities for credit blocked delivery document and reject reason. Once the delivery document has been checked, released, or rejected, a success message containing the document number is sent in the response.",
```

Comment:
Same

Reply:
Updated in [Update documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/11/commits/116c6665709fadd491c798aa5bb14999eefc57a1)